mappings/services.json

```diff
@@ -21,6 +21,24 @@
           "type": "stemmer",
           "name": "possessive_english"
         }
+      },
```

Comment:
Needs regeneration from the frameworks repo.

Reply:
Done